src/app/medicine/medicine.cpp

```diff
@@ -1,6 +1,6 @@
 #include "medicine.hpp"
 
-Medicine::Medicine(const std::string& name, const double price, const int amount, const int expiration_date,
+Medicine::Medicine(const std::string& name, const double& price, const int& amount, const int& expiration_date,
                    const std::pair<double, double>& storage_temperature, const std::vector<std::string>& composition)
     : Product(name, price, amount), expiration_date_{expiration_date}, storage_temperature_{storage_temperature},
       composition_{composition}
```

Comment:
Please don't add & to basic and lightweight type such as:
- int
- double
- enum types etc.